src/hooks/useVolumeControl.ts

```diff
@@ -0,0 +1,21 @@
+import { useCallback, useState } from "react";
+
+import { DEFAULT_NOTIFICATIONS_VOLUME } from "settings";
+
+export const useVolumeControl = (
+  defaultVolume: number = DEFAULT_NOTIFICATIONS_VOLUME
+) => {
+  const [volume, setVolume] = useState<number>(defaultVolume);
+
+  const volumeCallback = useCallback(
+    (value: number) => {
+      setVolume(value);
+    },
+    [setVolume]
+  );
+
+  return {
+    volume,
+    volumeCallback,
```

Comment:
i guess we can add `mute` actions to the hook as well to have everything in one place.

Reply:
Good idea for future PR